staff/almu-itur/pre-course/5-calculadora-html/app/app.js

```diff
@@ -0,0 +1,19 @@
+
+function calculator () {
+
+	var n1 = parseFloat(document.getElementById('n1').value);
+  	var n2 = parseFloat(document.getElementById('n2').value);
+  	var error = false;
+
+  	var operations = [n1 + n2, n1 - n2, n1 * n2, n1 / n2];
+	var resultsRounded = operations.map(function(numberVar) {
+       	if (!Number.isInteger(numberVar)) {
+        	numberVar = numberVar.toFixed(2);
```

Comment:
avoid re-assigning a value to a function argument (parameter), create a new variable  instead for that

Reply:
I thought it would be better to reuse